Project Lifecycle.md

```diff
@@ -0,0 +1,94 @@
+# Node.js Foundation Project Lifecycle
+
+## Project Definition
+
+The Node.js Foundation hosts several "Top Level Projects." These projects are autonomous from each other and governed by their own TC (Technical Committee) and chartered by the Node.js Foundation TSC.
+
+Projects are free to create "Working Groups" which are autonomous groups collaborating to fulfill a set of responsibilities. Working Groups are eventually chartered by the TC. The TSC also charters its own Working Groups.
+
+```
+ TSC
+  |
+  |-- Project A TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project TC)
+  |
+  |-- Project B TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project TC)
+  |
+  |-- Working Group A (Chartered by TSC)
+  |-- Working Group B (Chartered by TSC)
+```
+
+Both TLPs and TSC WGs may elect a representative to the TSC. TLPs and WGs with *incubation* status are not granted voting privileges on the TSC.
```

Comment:
Right, this is a little confusing. We have a bootstrapping problem with the TSC anyway though, so it's probably good that we have some members already there. Essentially what will happen is that members who just want to be involved in core can resign from the TSC (they will instead just be members of the Core TLP TC) as they will be quite bored :) This document describes how we admit new members from new TLPs which at some point in the future will probably represent all membership on the TSC.